frame/babe/src/lib.rs

```diff
@@ -782,7 +803,8 @@ impl<T: Config> OneSessionHandler<T::AccountId> for Module<T> {
 		where I: Iterator<Item=(&'a T::AccountId, AuthorityId)>
 	{
 		let authorities = validators.map(|(_, k)| (k, 1)).collect::<Vec<_>>();
-		Self::initialize_authorities(&authorities);
+		let epoch_config = BabeEpochConfiguration::default();
```

Comment:
We can't just use a default `BabeEpochConfiguration` when we were given one in genesis config.